### DISCUSSION TITLE
Add .jest-test-results.json to storybook-static

### DISCUSSION THREAD

**Discussion by rexkenley** [storybookjs/storybook#16276] [Help]
I used this guide to create test results with jest
https://storybook.js.org/addons/@storybook/addon-jest/

But how do I include the results json file to the storybook-static that was built with storybook-static?

**Comment by shilman** [comment]
https://storybook.js.org/docs/react/configure/images-and-assets#serving-static-files-via-storybook

**Reply by rexkenley** [reply]
I tried the -s option and do see the file in the storybook-static folder. However, how do I reference that file? This results with an error.

![image](https://user-images.githubusercontent.com/10548251/136448881-c9c7b1fa-a38d-4667-8535-c29446a5a37b.png)
![image](https://user-images.githubusercontent.com/10548251/136449183-6b5b8d04-50f5-4d0f-94d2-f4976888eb65.png)

**Reply by rexkenley** [reply]
@shilman 

The instructions work when running storybook, it's just the build-storybook that leaves out the json file. Is this a bug?
